experimental/dds/tree2/src/feature-libraries/editable-tree-2/README.md

```diff
@@ -46,12 +46,30 @@ There are four main usage modes for this API:
     This could be considered part of either reading API, but is being addressed separately here.
     APIs for receiving these changes take the form of registering callbacks for events.
     These events can be triggered by anything that impacts the content of the tree including local edits, remote edits, merges, rebasing local edits, transaction rollback, undo etc.
-    These can also be batched together, possibly eliminating intermediate states and reordering operations that don't impact eachother (like edits to different parts of the tree).
+    These can also be batched together, possibly eliminating intermediate states and reordering operations that don't impact each other (like edits to different parts of the tree).
     This means these event subscriptions should only be treated as changes to the content, and users of them should not attempt to ascribe meaning or providence to any particular event.
 
     Some control over these events is available via branching which provides snapshot isolation (among other things):
     this is done at a higher level than the tree API covered here but can be accessed via each tree entity's context property. (TODO: provide this access).
 
+To accommodate all of these at once in a single API surface, a few principals are followed in the API's design:
+
+-   Generic APIs suited for generic (non schema aware) tree readers form the base types of everything in the API.
+    These provide access to all content of the tree in a uniform manner, including objects for every field and node.
+    These are all traversable via iteration, and also can be walked along specific paths through the tree.
+    This for example motivates the include of the optional `.value` on TreeNode.
+-   Schema aware specializations of these types are provided which provide ergonomic access to optimized based on the specific kind of the node or field.
+    These APIs provide accessors which skip over levels of the tree (called "unboxing") that provide no additional information to a user that knows thew schema.
+    Since skipping over these layers is sometimes undesired (for example when needing to edit them), all APis which do this implicit "unboxing" have "boxed" alternatives that do not skip any layers of the tree.
```

Comment:
```suggestion
    Since skipping over these layers is sometimes undesired (for example when needing to edit them), all APIs which do this implicit "unboxing" have "boxed" alternatives that do not skip any layers of the tree.
```